__tests__/cron/personalizedDigest.ts

```diff
@@ -130,7 +132,9 @@ describe('personalizedDigest cron', () => {
       })),
     );
 
-    const logger = pino();
+    const logger = pino({
+      messageKey: 'message',
+    });
```

Comment:
Do we need multiple instances of logger or can we create a static one per file or maybe even per app? That way, we could centralise this configuration.